File: raft/src/main/java/com/example/raft/LogEntry.java

package com.example.raft;
import java.io.Serializable;

public class LogEntry implements Serializable {
    public int term;
    public String op;
    public int amount;

    public LogEntry(int term, String op, int amount) {
        this.term = term;
        this.op = op;
        this.amount = amount;
    }

    public String toString() {
        return "[" + term + " " + op + " " + amount + "]";
    }
}

File: raft/src/main/java/com/example/raft/LogEntry.kt
package com.example.raft

import java.io.Serializable

class LogEntry(var term: Int, var origin: Int, var op: String?, var amount: Int) : Serializable {
    override fun toString(): String {
        return "[term=" + term +
                ", origin=" + origin +
                ", op=" + op +
                ", amount=" + amount + "]"
    }
}
